acslib/ccure/search.py

```diff
@@ -16,7 +16,7 @@ def full_fuzz(term):
 
 
 def no_fuzz(term):
-    return f"{term}"
+    return term
```

Comment:
We want to enforce a str return no matter the input.

Reply:
Good call